src/Core/Commander/ManagerCommands.js

```diff
@@ -84,7 +89,7 @@ ManagerCommands.prototype.runCommand = function runCommand(command, queue, execu
         // We allow the scene to delay the update/repaint up to 100ms
         // to reduce CPU load (no need to perform an update on completion if we
         // know there's another one ending soon)
-        this.scene.notifyChange(100);
+        this.scene.notifyChange(100, true);
```

Comment:
`this.scene.notifyChange(100);` 
use `export const delayUpdate = 0`
and This value (100) may change depending on the computer